site/_docs/reference.md

```diff
@@ -3075,6 +3075,8 @@ In the following:
 | b | TO_CODE_POINTS(string)                         | Converts *string* to an array of integers that represent code points or extended ASCII character values
 | o p r h | TO_DATE(string, format)                    | Converts *string* to a date using the format *format*
 | o p r | TO_TIMESTAMP(string, format)               | Converts *string* to a timestamp using the format *format*
+| p | AGE(timestamp) | Returns the difference between the current timestamp and the specified timestamp
+| p | AGE(timestamp1, timestamp2) | Returns the difference between timestamp1 and timestamp2
```

Comment:
I saw the syntax `| b | TRUNC(numeric1 [, integer2 ])` near this line, and I think this form should be used.